app/src/main/kotlin/at/bitfire/davdroid/ui/AppSettingsScreen.kt

```diff
@@ -446,6 +458,39 @@ fun AppSettings_Security(
     )
 }
 
+@Composable
+fun DistrustSystemCertificatesAlertDialog(
+    onDistrustSystemCertsRequested: () -> Unit,
+    onDismissRequested: () -> Unit,
+) {
+    AlertDialog(
+        onDismissRequest = onDismissRequested,
+        title = { Text(stringResource(R.string.app_settings_distrust_system_certs_dialog_title)) },
```

Comment:
I'd use the option title here again ("Distrust system certificates") to make clear what the dialog is about.

May we can use an attention sign as [icon](https://m3.material.io/components/dialogs/guidelines#12bb8f46-5ccd-40ca-99bb-e9cfcd6ce228) instead of "Attention"?